pkg/sql/distsql_plan_bulk.go

```diff
@@ -46,9 +47,26 @@ func (dsp *DistSQLPlanner) SetupAllNodesPlanningWithOracle(
 	localityFilter roachpb.Locality,
 ) (*PlanningCtx, []base.SQLInstanceID, error) {
 	if dsp.codec.ForSystemTenant() {
-		return dsp.setupAllNodesPlanningSystem(ctx, evalCtx, execCfg, oracle, localityFilter)
+		return dsp.setupAllNodesPlanningSystem(ctx, evalCtx, execCfg, oracle, localityFilter, nil /*txn*/)
 	}
-	return dsp.setupAllNodesPlanningTenant(ctx, evalCtx, execCfg, oracle, localityFilter)
+	return dsp.setupAllNodesPlanningTenant(ctx, evalCtx, execCfg, oracle, localityFilter, nil /*txn*/)
+}
+
+// SetupAllNodesPlanningWithOracleAndFollowers is like
+// SetupAllNodesPlanningWithOracle but with a txn param that if non-nil allows
+// the flow to plan on followers.
+func (dsp *DistSQLPlanner) SetupAllNodesPlanningWithOracleAndFollowers(
```

Comment:
would be nice to have tests for this... should be possible w/ injectable oracle implementations.  
TODO probably fine, but we really need to know for sure that this
stuff works correctly.